projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-flag/read-case-flag-field.component.spec.ts

```diff
@@ -375,6 +387,9 @@ describe('ReadCaseFlagFieldComponent', () => {
     fixture = TestBed.createComponent(ReadCaseFlagFieldComponent);
     component = fixture.componentInstance;
     component.caseField = flagLauncher1CaseField;
+    router = TestBed.get(Router);
```

Comment:
@MateiHMCTS - please change this (and the line below) to use `inject` rather than the deprecated `get`